### DISCUSSION TITLE
SQL Backup is showing physical_device_name {C1721FC4-81BB-4CE1-945D-434CBC989BF8}31

### DISCUSSION THREAD

**Discussion by MarcoBus** [olahallengren/sql-server-maintenance-solution#795] [Questions & Answers]
Hi.

can someone explain. Why some databases show {C1721FC4-81BB-4CE1-945D-434CBC989BF8}31 as physical_device_name instead of the physical path on the server. this only happens with some databases. 
I have also noticed that the backup job does not create the directory for the database under the backup directory.

I use Ola Hallengren MaintenanceSolution script.

**Comment by griffitmatt** [comment]
That value you are seeing suggests a third party backup tool is being used to perform the database backups (virtual device). If you check the device_type (l column in the backupmediafamily table you will most likely the value (7) which means Virtual device.

**Comment by MarcoBus** [comment]
@griffitmatt 
Thanks for the reply. Saw that I had forgotten to uncheck so that Veeam doesn’t take SQL backup on the server. Have now changed so that it only takes VM backup and SQL the DB backups.
Question do I just clear the backup history so the script starts taking backups again.

**Comment by MarcoBus** [comment]
I have cleared the backup history.
and now I only use Ola Hallengren's script to run backups for the databases.
but all the db that veeam backed up are not included in the job. Does anyone know why.

**Reply by griffitmatt** [reply]
Are you getting any output in the job history/log file? Anything in the sql server error log?
